### DISCUSSION TITLE
arm版本部署集群milvus-v2.4.17版本报这个错

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38727] [Announcement]
![image](https://github.com/user-attachments/assets/2a4239d6-e967-4021-8049-b61dfa266d26)

![image](https://github.com/user-attachments/assets/bfb4cff4-cc23-46ab-ba5a-ec78936dcff7)

![image](https://github.com/user-attachments/assets/b388cb65-86f1-4aaf-ba2e-5f49d9110c6a)
有没有知道的大佬帮忙看下 谢谢，

**Comment by presburger** [comment]
@jhyever What model is the ARM CPU in the server?
